src/Tbody.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import fakeBookings from "./fakeBookings.js";
+import moment from "moment";
+const Tbody = () => {
+  return (
+    <tbody>
+      {fakeBookings.map(booker => {
+        console.log(fakeBookings);
```

Comment:
I think you forgot some debugging code in here.

Reply:
yes